Build/run.sh

```diff
@@ -56,7 +56,7 @@ while [[ $# -gt 0 ]]; do
         ;;
     esac
 done
-CONTNAME="$(whoami)-${NAME}${BRANCH}"
+CONTNAME="$(whoami)-${NAME}_${BRANCH}"
```

Comment:
If the branch is empty, it would leave an unsightly underline tail at the end. Please fix.